d2bs/kolbot/default.dbj

```diff
@@ -34,6 +34,8 @@ function main() {
 		delay(50);
 	}
 
+	clearAllEvents() // remove any event listeners from game crash
```

Comment:
nit: add semicolon

Reply:
I have added the semi-colon and will update the PR